app/src/ui/diff/text-diff.tsx

```diff
@@ -389,8 +381,13 @@ export class TextDiff extends React.Component<ITextDiffProps, ITextDiffState> {
       return
     }
 
-    const { file, repository } = this.props
-    const { contents: currentContents, diff: currentDiff } = this.state.document
+    const contents = this.props.fileContents
+
+    if (contents === null) {
```

Comment:
Same comment here as in side-by-side about not coupling expansion to highlighting